cds_ils/importer/providers/ebl/importer.py

```diff
@@ -14,4 +14,4 @@ class EBLImporter(Importer):
     """EBL importer class."""
 
     EITEM_OPEN_ACCESS = False
-    EITEM_URLS_LOGIN_REQUIRED = False
+    EITEM_URLS_LOGIN_REQUIRED = True
```

Comment:
Is this the default value? Does it mean that it was not correct before?

Reply:
its the default value, the library requested this change after testing, it seems it was wrong, same in the comment below